### PR TITLE
Release 1.0.0-beta.15.1

### PR DESCRIPTION
**Pull request by dmtrKovalenko**

### CODE REVIEW

File: lib/package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
   },
   "peerDependencies": {
     "classnames": "^2.2.5",
-    "material-ui": "^1.0.0-beta.31",
+    "material-ui": "^1.0.0-beta.33",
```
Comment (cherniavskii) [inline lib/package.json]: why 33?

Comment (cherniavskii) [reply lib/package.json]: isn't it compatible with 32 too?

Comment (dmtrKovalenko) [reply lib/package.json]: Oh ( 
Yes but I would not publish additional release :)

Comment (cherniavskii) [reply lib/package.json]: it would be great to have a possibility to review release PRs *before* release

Comment (dmtrKovalenko) [reply lib/package.json]: Sorry :) 
Just think that it would be just fast release with the fixes) 
Will assign pull request and wait review. My fault.

Comment (cherniavskii) [reply lib/package.json]: no worries :)

**Pull request merged**